src/guide/code-owners.md

```diff
@@ -4,41 +4,44 @@ This list is intended to make it easier to identify which email list to include
 
 ## Area mailing lists
 
-* Build: [`build-dev@openjdk.org`](mailto:build-dev@openjdk.org)
+* Build: [`build-dev`](https://mail.openjdk.org/mailman/listinfo/build-dev)
 * Client
-  * Client Libs: [`client-libs-dev@openjdk.org`](mailto:client-libs-dev@openjdk.org)
-  * Java FX: [`openjfx-dev@openjdk.org`](mailto:openjfx-dev@openjdk.org)
-  * jpackage: [`core-libs-dev@openjdk.org`](mailto:core-libs-dev@openjdk.org)
-* Core Libs: [`core-libs-dev@openjdk.org`](mailto:core-libs-dev@openjdk.org)
-  * Net: [`net-dev@openjdk.org`](mailto:net-dev@openjdk.org)
-  * NIO: [`nio-dev@openjdk.org`](mailto:nio-dev@openjdk.org)
-* HotSpot: [`hotspot-dev@openjdk.org`](mailto:hotspot-dev@openjdk.org)
-  * Compiler: [`hotspot-compiler-dev@openjdk.org`](mailto:hotspot-compiler-dev@openjdk.org)
-  * GC: [`hotspot-gc-dev@openjdk.org`](mailto:hotspot-gc-dev@openjdk.org)
-  * Runtime: [`hotspot-runtime-dev@openjdk.org`](mailto:hotspot-runtime-dev@openjdk.org)
-  * JFR: [`hotspot-jfr-dev@openjdk.org`](mailto:hotspot-jfr-dev@openjdk.org)
-  * Serviceability: [`serviceability-dev@openjdk.org`](mailto:serviceability-dev@openjdk.org)
-* I18n: [`i18n-dev@openjdk.org`](mailto:i18n-dev@openjdk.org)
-* LangTools
-  * Amber Project: [`amber-dev@openjdk.org`](mailto:amber-dev@openjdk.org)
-  * Javac: [`compiler-dev@openjdk.org`](mailto:compiler-dev@openjdk.org)
-  * Javadoc: [`javadoc-dev@openjdk.org`](mailto:javadoc-dev@openjdk.org)
-  * JShell: [`kulla-dev@openjdk.org`](mailto:kulla-dev@openjdk.org)
-  * Panama Project: [`panama-dev@openjdk.org`](mailto:panama-dev@openjdk.org)
-  * Valhalla Project: [`valhalla-dev@openjdk.org`](mailto:valhalla-dev@openjdk.org)
-* Security: [`security-dev@openjdk.org`](mailto:security-dev@openjdk.org)
+  * Client Libs: [`client-libs-dev`](https://mail.openjdk.org/mailman/listinfo/client-libs-dev)
+  * Project Wakefield: [`wakefield-dev`](https://mail.openjdk.org/mailman/listinfo/wakefield-dev)
```

Comment:
Thinking about it I do agree with you. These are in the list because they were sent to me as something that should be on the list. But really, if there's no directory in the list below that refers to the area, then the mailing list doesn't really fit in this list. There are other places to find mailing lists for different projects.